mediachain/indexer/mc_simpleclient.py

```diff
@@ -372,19 +373,24 @@ def get_objects(self,
             
             try:
                 started = False
-                with self.transactor.canonical_stream(catchup=catchup_blockchain, timeout=timeout) as stream:
-                    for ref, obj in stream:
+                with self.transactor.canonical_stream(
+                    catchup=catchup_blockchain,
+                    last_known_block_ref=last_known_block_ref,
+                    timeout=timeout
+                ) as stream:
+                    for obj_info in stream:
+                        obj = obj_info['record']
 
                         if (start_id is not False) and (not started):
-                            if obj['data']['_id'] == start_id:
+                            if obj['meta']['data']['_id'] == start_id:
                                 started = True
                             else:
                                 continue
 
-                        yield ref, obj
+                        yield obj_info
```

Comment:
yeah, the `get_artefacts` and `get_entities` functions just pass through this return value unchanged, and only `get_artefacts` is used ATM.  I updated `receive_blockchain_into_indexer` to unpack the dictionary instead of the tuple. 

The only other place its used is in the debug `tail_blockchain` method, which just prints without unpacking anything